CMakeLists.txt

```
@@ -0,0 +1,46 @@
+project(x16emu)
+cmake_minimum_required(VERSION 3.0)
```

These 2 lines should be switched.
First call `cmake_minimum_required`, then `project`.

---

Also, I would do
```suggestion
project(x16emu LANGUAGES C)
```
since CXX is not used.